### PR TITLE
Create ranks contracts

### PR DESCRIPTION
**Pull request by t0ms0n00**
How to test the interaction:
- install metamask browser extension
- `cd backend && npx ganache-cli --mnemonic "lighten crowd lavish spectacular bustling saw tasteless nauseating panoramic mixed absorbing past"`
- in a new terminal: `truffle console`, then `migrate`
- in a new terminal: `cd frontend && npm start`
- from ganache copy the first private key and import it to metamask
- connect a wallet on the home screen
- create ranks
- go to create events, the ranks should be listed in dropdown

### CODE REVIEW

File: frontend/src/utils/web3-helpers.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+export enum DynamicContracts {
+  RANKS = "ranks",
+}
+
+const ethereum = window.ethereum;
+//@ts-ignore
+const web3 = new Web3(ethereum);
+
+const formatContract = (contract: { [key: string]: any }) => {
```
Comment (robertkbk) [inline frontend/src/utils/web3-helpers.ts:25]: `
```suggestion
const formatContract = (contract: Record<string, unknown>) => {
```

File: frontend/src/screens/AdminEventsPanel.tsx [hunk 2]
```diff
@@ -109,4 +155,25 @@ const StyledInput = styled.input`
   }
 `;
 
-export default AdminPanel;
+const StyledSelect = styled.select`
```
[the remark this answers is not in the document]
Comment (t0ms0n00) [reply frontend/src/screens/AdminEventsPanel.tsx:158]: @delekta for me we can switch to this wdyt?

**Review by delekta — APPROVED** [review]
LGTM 🌴

File: frontend/src/screens/AdminEventsPanel.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (delekta) [reply frontend/src/screens/AdminEventsPanel.tsx:152]: If you want, but I don't think it is needed since almost everything is done

**Pull request merged**